nix/drun.nix

```diff
@@ -1,83 +1,81 @@
-pkgs:
-{ drun =
-    pkgs.rustPlatform_moz_stable.buildRustPackage {
-      name = "drun";
+pkgs: pkgs.rustPlatform-stable.buildRustPackage {
+  pname = "drun";
+  version = builtins.substring 0 7 pkgs.sources.ic-src.rev;
 
-      src = pkgs.sources.ic;
+  src = pkgs.sources.ic-src;
 
-      # update this after bumping the dfinity/ic pin.
-      # 1. change the hash to something arbitrary (e.g. flip one digit to 0 or use `pkgs.lib.fakeSha256`)
-      # 2. run nix-build -A drun nix/
-      # 3. copy the “expected” hash from the output into this file
-      # 4. commit and push
-      #
-      # To automate this, .github/workflows/update-hash.yml has been
-      # installed. You will normally not be bothered to perform
-      # the command therein manually.
+  # update this after bumping the dfinity/ic pin.
+  # 1. change the hash to something arbitrary (e.g. flip one digit to 0 or use `pkgs.lib.fakeSha256`)
+  # 2. run nix-build -A drun nix/
+  # 3. copy the “expected” hash from the output into this file
+  # 4. commit and push
+  #
+  # To automate this, .github/workflows/update-hash.yml has been
```

Comment:
IIRC, above you say `update-hash.yml` is obsolete. I think this comment can go.

Reply:
Or maybe not?

Reply:
No update-hash.yml is still used to update the hashes in drun.